src/megameklab/com/util/RefreshListener.java

```diff
@@ -1,36 +1,36 @@
-/*
- * MegaMekLab - Copyright (C) 2008
- *
- * Original author - jtighe (torren@users.sourceforge.net)
- *
- * This program is free software; you can redistribute it and/or modify it
- * under the terms of the GNU General Public License as published by the Free
- * Software Foundation; either version 2 of the License, or (at your option)
- * any later version.
- *
- * This program is distributed in the hope that it will be useful, but
- * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
- * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
- * for more details.
- */
-
-package megameklab.com.util;
-
-import java.util.EventListener;
-
-public interface RefreshListener extends EventListener{
-    public void refreshHeader();
-    public void refreshStatus();
-    public void refreshAll();
-    public void refreshStructure();
-    public void refreshArmor();
-    public void refreshWeapons();
-    public void refreshEquipment();
-    public void refreshTransport();
-    public void refreshBuild();
-    public void refreshPreview();
-
-    // Refreshers for just one thing on a tab
-    public void refreshSummary();
-    public void refreshEquipmentTable();
+/*
+ * MegaMekLab - Copyright (C) 2008
+ *
+ * Original author - jtighe (torren@users.sourceforge.net)
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+ * for more details.
+ */
+
+package megameklab.com.util;
+
+import java.util.EventListener;
+
+public interface RefreshListener extends EventListener{
```

Comment:
Why does this entire file show as changed? Spacing/file save issues?

Reply:
Usually that means the line endings were CR/LF and they got changed to LF. GitHub lets you hide whitespace changes when you view the diff so you can see what actually changed.